bsp/imx/imx93-11x11-lpddr4x-evk-preempt-rt.scc

```diff
@@ -0,0 +1,9 @@
+define KMACHINE imx93-11x11-lpddr4x-evk
+define KARCH aarch64
+define KTYPE preempt-rt
+
+include ktypes/standard/standard.scc
+
+include imx93-11x11-lpddr4x-evk.scc
+
+include ktypes/preempt-rt/preempt-rt.scc
```

Comment:
We should probably drop preempt from this commit, as it needs to be tested and so on first.